frontend/test/app/components/ExperimentTourDialog-test.js

```diff
@@ -63,8 +62,9 @@ describe('app/components/ExperimentTourDialog', () => {
     const expectedTourStep = props.experiment.tour_steps[1];
     expect(subject.find('.tour-image > img').prop('src'))
       .to.equal(expectedTourStep.image);
-    expect(subject.find('.tour-text').html())
-      .to.include(expectedTourStep.copy);
+    // XXX fluent-react
+    // expect(subject.find('.tour-text').html())
+    //   .to.include(expectedTourStep.copy);
```

Comment:
The resulting HTML won't contain the expected copy. Perhaps the test should check if the `id` set on `Localized` is correct?